cpp/models/ode_secir/model.h

```diff
@@ -124,41 +116,46 @@ class Model : public CompartmentalModel<InfectionState, Populations<AgeGroup, In
                                  (params.get<RelativeTransmissionNoSymptoms>()[j] * pop[INSj] +
                                   riskFromInfectedSymptomatic * pop[ISyj]);
 
-                dydt[Si] -= dummy_S;
-                dydt[Ei] += dummy_S;
+                // Susceptible -> Exposed
+                flows[get_flow_index<InfectionState::Susceptible, InfectionState::Exposed>({j})] += dummy_S;
             }
 
             // ICU capacity shortage is close
             double criticalPerSevereAdjusted =
                 smoother_cosine(icu_occupancy, 0.90 * params.get<ICUCapacity>(), params.get<ICUCapacity>(),
                                 params.get<CriticalPerSevere>()[i], 0);
 
-            double deathsPerSevereAdjusted = params.get<CriticalPerSevere>()[i] - criticalPerSevereAdjusted;
-
-            dydt[Ei] -= rateE * y[Ei]; // only exchange of E and INS done here
-            dydt[INSi] = rateE * y[Ei] - rateINS * y[INSi];
-            dydt[ISyi] = (1 - params.get<RecoveredPerInfectedNoSymptoms>()[i]) * rateINS * y[INSi] -
-                         (1 / params.get<TimeInfectedSymptoms>()[i]) * y[ISyi];
-            dydt[ISevi] = params.get<SeverePerInfectedSymptoms>()[i] / params.get<TimeInfectedSymptoms>()[i] * y[ISyi] -
-                          (1 / params.get<TimeInfectedSevere>()[i]) * y[ISevi];
-            dydt[ICri] = -(1 / params.get<TimeInfectedCritical>()[i]) * y[ICri];
-            // add flow from hosp to icu according to potentially adjusted probability due to ICU limits
-            dydt[ICri] += criticalPerSevereAdjusted / params.get<TimeInfectedSevere>()[i] * y[ISevi];
-
-            dydt[Ri] =
-                params.get<RecoveredPerInfectedNoSymptoms>()[i] * rateINS * y[INSi] +
-                (1 - params.get<SeverePerInfectedSymptoms>()[i]) / params.get<TimeInfectedSymptoms>()[i] * y[ISyi] +
-                (1 - params.get<CriticalPerSevere>()[i]) / params.get<TimeInfectedSevere>()[i] * y[ISevi] +
+            // double deathsPerSevereAdjusted = params.get<CriticalPerSevere>()[i] - criticalPerSevereAdjusted;
```

Comment:
We need a test that this fails next time!

Reply:
Interestingly (possibly even consciously ??) by the choice of the parameters 
      `criticalPerSevereAdjusted = 0.25` and` CriticalPerSevere = 0.25`. Therefore `deathsPerSevereAdjusted ` was always 0. 
Test now fixed.